src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -133,8 +135,8 @@ public function terminate(Request $request, Response $response)
             return;
         }
 
-        if ($this->getHttpKernel() instanceof TerminableInterface) {
-            $this->getHttpKernel()->terminate($request, $response);
+        if ($this->httpKernel instanceof TerminableInterface) {
```

Comment:
Agreed. Should I remove the property altogether then? This way we'd have one call in `handle` and one call here